supabase/models/messaging/getAdditionalConversationDetails.ts

```diff
@@ -0,0 +1,19 @@
+import newClient from '@/supabase/utils/newClient';
+
+export default async function getAdditionalConversationDetails(
+  user_id: string | undefined,
+  conversation_id: number | undefined
+) {
+  if (user_id && conversation_id) {
+    const supabase = newClient();
+    const { data } = await supabase
+      .from('user_conversations')
+      .select('partner_id, item_id')
+      .eq('user_id', user_id)
+      .eq('conversation_id', conversation_id)
+      .single();
+    return data;
+  } else {
+    console.error('userID or Conversation ID undefined');
+  }
```

Comment:
This is a job TypeScript should be covering for us.

There's no express need for us to make the props `| undefined` since we don't want to be able to call this util on any values that are potentially undefined, so then we wouldn't need this `else` block